src/Reservation.php

```diff
@@ -188,7 +188,7 @@ public static function handleAddForm(array $input): void
             $input['users_id'] = Session::getLoginUserID();
         }
         if (
-            !Session::haveRight("reservation", UPDATE)
+            !Session::haveRight("reservation", ReservationItem::RESERVEANITEM)
             && Session::getLoginUserID() !== $input['users_id']
```

Comment:
Maybe this line can be removed. Indeed, this line would permit to anyone to reserve an item, even when they do not have the `RESERVEANITEM` right.

```suggestion
```